pages/blog/blog.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/globals.scss';
+
+.container {
+	padding: $space-md $space-xxxl;
+	margin-bottom: $space-xxl;
+
+	p {
+		font-family: $primaryTextFont;
+	}
+
+	header {
+		position: relative;
+		margin-bottom: $space-lg;
+
+		h3 {
+			font-family: $secondaryHeaderFont;
+			font-size: $f3;
+			margin-bottom: $space-xs;
+		}
+
+		.socialContainer {
+			position: absolute;
+			left: -$space-xxl;
+			text-align: center;
+
+			li {
+				margin: $space-md 0;
+			}
+		}
+	}
+
+	.blog {
+		display: inline-block;
+
+		p {
```

Comment:
lets move this under the `blogThumbnail` styles to keep consistent with the `jsx` structure